common/filetoolkitwithundo.cpp

```diff
@@ -174,6 +174,35 @@ bool FileToolkitWithUndo::syncOverlayMount(const std::string &lower,
     return RecursiveCopy::getInstance().copy(upper, lower);
 }
 
+bool FileToolkitWithUndo::tmpfsMount(const std::string dst, const int maxSize)
+{
+    std::string fstype = "tmpfs";
+    unsigned long flags = 0;
+    std::unique_ptr<CreateDir> createDirInstance = std::unique_ptr<CreateDir>(new CreateDir());
+
+    if (!createDirInstance->createDirectory(dst)) {
+        log_error() << "Failed to create " << dst << " directory for tmpfs mount";
+        return false;
+    }
+
+    std::string mountoptions = logging::StringBuilder() << "size=" << maxSize;
```

Comment:
This can be done with std::to_string as well I think. No need to change now, just a thought.